### ISSUE TITLE
Were the pathologies verified on CT?

### ISSUE THREAD

**Issue by cyrilzakka** [Deepwise-AILab/ChestX-Det-Dataset#6]
Looking at some of the annotations, especially nodules, were the findings confirmed on CT? Otherwise they can be impossible to detect, especially after eyeballing a few images. Would it be possible to have more information on this data?

**Comment by david-rohrschneider** [comment]
I totally agree to @cyrilzakka. The models i am trying to train cannot detect the nodules either, since they have no visual contours on the images
